src/Adapter/OrderMessage/CommandHandler/EditOrderMessageHandler.php

```diff
@@ -69,4 +73,26 @@ public function handle(EditOrderMessageCommand $command): void
             throw new OrderMessageException(sprintf('Failed to update order message with id "%s"', $command->getOrderMessageId()->getValue()), 0, $e);
         }
     }
+
+    private function assertNameIsNotAlreadyUsed(EditOrderMessageCommand $command): void
```

Comment:
I dont like the repetition of this method. Can it be in the abstract ?

Reply:
No the `assert` name is logical but the behavior is different inside

Reply:
My bad.
So in AddOrderMessage we check that message title doesnt exist at all
In EditOrderMessage that it doesn't exist in some other orderMessage